mod/wet4/views/default/page/elements/message_dd_list.php

```diff
@@ -20,7 +20,7 @@
 		echo $no_results();
 		return;
 	}
-	echo "<p class='elgg-no-results message-dd-no-results'>$no_results</p>";
+	echo "<p class='elgg-no-results message-dd-no-results' aria-live='assertive'>$no_results</p>";
```

Comment:
This paragraphe is in a dropdown menu generate by a ajax call. If aria-live is not there, the screen reader does not read the paragraphe.